examples/src/content/jcr_root/content/core-components-examples/library/container/_jcr_content/image/file.dir/.content.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
The image is copied from the carousel, we will need a different and more suggestive image for the container.